tests/test_end_to_end.py

```diff
@@ -212,8 +213,12 @@ def test_all_blocks_page_to_markdown(tmp_path):
     assert "``` javascript\nCode Block\n```" in document_as_markdown
     assert lines.count("This is a synced block.") == 2
     assert "This is a synced block from another page." in lines
+    assert "This page is not shared with the integration." in lines
+
     assert all(column_strings_in_lines) or (column_string in lines)
 
+
```

Comment:
Flake8 will catch this, but these new lines are not pep8-compliant.

Reply:
Yes, thanks.